easybuild/easyconfigs/s/Satsuma2/Satsuma2-2022-03-04-GCC-11.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'Satsuma2'
+local_commit = '37c5f38'
+version = '2022-03-04'
```

Comment:
Hmm, using `20220304` would fit better with other easyconfigs where we use a datestamp as a workaround for the lack of a proper version, so I'll change that...